symfony/plugins/orangehrmMarketPlacePlugin/config/config.yml

```diff
@@ -0,0 +1 @@
+getAddons: 'https://jsonplaceholder.typicode.com/todos/'
```

Comment:
What is this external url?

Reply:
this is just to developing API request used for test purpose